docs/source/models/cause.rst

```diff
@@ -337,14 +336,38 @@ Mortality Impacts
 Cause-Specific Mortality
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+The cause-specific mortality rate is the mortality rate from a specified cause for a population. 
+The numerator is the number of deaths attributed to a specific cause. The denominator remains the 
+size of the population. In GBD, cause-specific estimates by country, year, age, sex.
+
+  For example, 112 measles deaths in country A during calendar year 2016. 1010,623 estimated 2016 mid-year 
+  population. 11.08 measles deaths per 100,000 population (person-years at risk) during calendar year 
+  2016 in country A. 
+
 Excess Mortality
 ^^^^^^^^^^^^^^^^
 
+The excess mortality is obtained by subtracting the expected mortality from the observed mortality.
+The excess mortality can be interpreted as the mortality directly or indirectly related to the disease of 
+interest, if one excludes all other causes of death. It is the extra number of deaths observed, beyond that 
+expected, per person-years.   
```

Comment:
I have found people are very confused about this. Perhaps belabor the point by discussing two populations that are matched in all ways, except for one is with-condition and the other is susceptible.  Also remind the reader that the denominator here is person-years among those with condition, not the total population.